index.html

```diff
@@ -1,15 +1,15 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="IE=edge">
-    <title>Lightbeam</title>
-    <meta name="description" content="Lightbeam is a Firefox add-on that uses interactive visualizations to show you the relationships between third parties and the sites you visit.">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
-    <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
-  </head>
-  <body>
-  <h1>Lightbeam</h1>
-  </body>
-</html>
+<!doctype html>

```

Comment:
This might be line endings perhaps? You can reset a file to a branch in future if this happens again. Lets ignore for now though.